lib/index.js

```diff
@@ -1,69 +1,70 @@
 "use strict";
 
 const async = require("async");
-const _ = require("lodash");
 const fs = require("fs-extra");
+const https = require("https");
 const os = require("os");
 const path = require("path");
-const https = require("https");
 
 const App = require("./app");
 
-const S3rver = function(options) {
-  this.options = _.defaults({}, options, S3rver.defaultOptions);
-};
-
-S3rver.prototype.resetFs = function(callback) {
-  const { directory } = this.options;
-  fs.readdir(directory, function(err, buckets) {
-    if (err) return callback(err);
-    async.eachSeries(
-      buckets,
-      function(bucket, callback) {
-        fs.remove(path.join(directory, bucket), callback);
-      },
-      callback
-    );
-  });
-};
+class S3rver {
+  constructor(options) {
+    this.options = Object.assign({}, S3rver.defaultOptions, options);
```

Comment:
@specialkk This is not the same as the previous code. An `undefined` property in `options` would override `defaultOptions` here, whereas the default would be used in the previous code.

I'm assuming you want to have fewer dependencies, but there are benefits to using `lodash` that outweigh the cost of depending on it in 99% of the cases (even `forEach` – lodash is faster than native). Destructure the methods if you want to get rid of the ugly `_.` syntax but I'd encourage you to use `lodash` wherever it makes sense, and not to remove lodash method calls with prejudice.

Reply:
You're right, I'll revert the `Object.assign`. Nothing against using lodash. I think the only instances I ended up replacing it was when `_.map` was used.